APMrover2/commands_logic.cpp

```diff
@@ -104,7 +104,14 @@ bool Rover::start_command(const AP_Mission::Mission_Command& cmd)
 // exit_mission - callback function called from ap-mission when the mission has completed
 void Rover::exit_mission()
 {
+    // play a tone
+    AP_Notify::events.mission_complete = 1;
+    // send message
     gcs().send_text(MAV_SEVERITY_NOTICE, "Mission Complete");
+
+    if (g2.mis_done_behave == MIS_DONE_BEHAVE_LOITER && set_mode(mode_loiter, MODE_REASON_MISSION_END)) {
```

Comment:
If the current mode is the hold mode and BEHAVE is specified as Reuta, is it a recognition that "Switch from hold mode to Reuta mode"?

Reply:
I think the answer is YES. It would be LOITER after AUTO mission done.